docs/feature/subnet-scarcity/proposal.md

```diff
@@ -49,27 +49,18 @@ DNC (which maintains the state of the Subnet in its database) will cache the res
 per Subnet. DNC will also expose an API to query $R$ of the Subnet, the `SubnetState` API.
 
 #### [[1-2]](phase-1/2-exhaustion.md) Subnet Exhaustion is calculated by DNC-RC
-DNC-RC will poll the `SubnetState` API to periodically check the Subnet utilization. DNC-RC will be configured with a lower and upper threshold ( $t$ and $T$ ) as fractions of the Subnet capacity $Q$. If the Subnet utilization crosses the upper threshold, DNC-RC will consider the Subnet "exhausted". If the Subnet utilization falls below the lower threshold, DNC-RC will consider the Subnet "not exhausted". Two values are necessary to induce hysteresis and avoid continous oscillation between the two states.
-
-$$
-E = !E \text{(toggle exhaustion) when}\begin{cases}
-R \gt T \times Q &\text{if not exhausted}\\
-R \lt t \times Q &\text{if exhausted}
-\end{cases}
-$$
-
-If the Subnet is exhausted, DNC-RC will write an additional per-subnet CRD, the [`ClusterSubnetState`](https://github.com/Azure/azure-container-networking/blob/master/crd/clustersubnetstate/api/v1alpha1/clustersubnetstate.go), with a Status of `exhausted=true`. When the Subnet is not exhausted, DNC-RC will write the Status as `exhausted=false`.
+DNC-RC will DNC's SubnetState API on an interval to check the Subnet Utilization. If the Subnet Utilization crosses some configurable lower and upper thresholds, RC will consider that Subnet un-exhausted or exhausted, respectively, and will write the exhaustion state to the ClusterSubnet CRD.
```

Comment:
`DNC-RC will request DNC's SubnetState API on a timed loop to check the Subnet Utilization. If the Subnet Utilization crosses some configurable lower and upper thresholds, RC will consider that Subnet is un-exhausted or exhausted, respectively, and will write the exhaustion state to the ClusterSubnet CRD.'

nit improvements suggested

Reply:
fixed